src/main/java/com/leverx/learningmanagementsystem/core/security/converter/CustomTokenAuthenticationConverter.java

```diff
@@ -0,0 +1,66 @@
+package com.leverx.learningmanagementsystem.core.security.converter;
```

Comment:
I recommend to place this converter in `com.leverx.learningmanagementsystem.btp.xsuaa.converter` as it's XSUAA-specific converter (other auth servers can generate JWT with another structure and we will have to implement dedicated converter for it)